src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  select: string,
+  setSelect:(value: string) => void;
+  todos: Todo[];
+  lengTodos: number,
+  handleDeleteTodoCompleted: () => void;
+};
+
+export const TodoFooter: React.FC<Props> = (
+  {
+    setSelect,
+    lengTodos,
+    select,
+    handleDeleteTodoCompleted,
+    todos,
+  },
+) => {
+  const buttonClear = todos.filter(todo => todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {lengTodos}
+        {' '}
+        items left
```

Comment:
Mentor means to use interpolation 
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/70009205/97366923-2a44-4dc6-a66e-0017a4bed45f)
```suggestion
        {`${lengTodos}  item${lengTodos > 1 ? 's' : '' } left`}
```